Source/Core/Core/Movie.cpp

```diff
@@ -369,9 +370,9 @@ void SignalDiscChange(const std::string& new_path)
     constexpr size_t maximum_length = sizeof(DTMHeader::discChange);
     if (filename.length() > maximum_length)
     {
-      PanicAlertT("The disc change to \"%s\" could not be saved in the .dtm file.\n"
-                  "The filename of the disc image must not be longer than 40 characters.",
-                  filename.c_str());
+      PanicAlertFmtT("The disc change to \"{0}\" could not be saved in the .dtm file.\n"
```

Comment:
Shouldn't this be `{}` to be consistent with the other strings with only one parameter.